bugbug/models/invalid_compatibility_report.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+
+import xgboost
+from sklearn.compose import ColumnTransformer
+from sklearn.pipeline import Pipeline
+
+from bugbug import feature_cleanup, issue_features, utils
+from bugbug.model import IssueModel
+
+logger = logging.getLogger(__name__)
+
+
+class InvalidCompatibilityReportModel(IssueModel):
+    def __init__(self, lemmatization=False):
+        super().__init__(
+            owner="webcompat", repo="web-bugs", lemmatization=lemmatization
+        )
+
+        self.calculate_importance = False
+
+        feature_extractors = []
+
+        cleanup_functions = [feature_cleanup.CleanCompatibilityReportDescription()]
+
+        self.extraction_pipeline = Pipeline(
+            [
+                (
+                    "report_extractor",
+                    issue_features.IssueExtractor(
+                        feature_extractors, cleanup_functions, rollback=False
+                    ),
+                ),
+                (
+                    "union",
+                    ColumnTransformer(
+                        [
+                            (
+                                "first_comment",
+                                self.text_vectorizer(min_df=0.0001),
+                                "first_comment",
+                            ),
+                        ]
+                    ),
+                ),
+            ]
+        )
+
+        self.clf = xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
+
+    def get_labels(self):
+        classes = {}
+        for issue in self.github.get_issues():
+            # Skip issues with empty title or body
+            if issue["title"] or issue["body"]:
```

Comment:
Nit: to increase readability, you could make this:
```Python
if not issue["title"] and not issue["body"]:
    continue
```
so the indentation for the rest of the code can be reduced

Reply:
Sounds good. I guess it should be `or` i.e. `if not issue["title"] or not issue["body"]:` 
It's mostly for issues like https://github.com/webcompat/web-bugs/issues/109865, where user edited the content. It has a title, but no description, so it's ok to skip it.